juniper_official/chassis/check-routing-engine-temperature.rule

```diff
@@ -37,10 +37,16 @@ healthbot {
                  synopsis "routing-engine temperature KPI";
                  description "open-config sensor to collect telemetry data from network device";
                  open-config {
-                     sensor-name /components/;
+                     sensor-name /components/component;
                      frequency 60s;
                  }
              }
+             sensor components-oc-others {
+                 open-config {
+                     sensor-name /components/;
```

Comment:
This also should be /components/component

Reply:
The sensor "components-oc-others"" will be removed as RE temperature path will be "temperature" now across all models.